src/File.roc

```diff
@@ -39,7 +44,7 @@ write = \path, val, fmt ->
 ## ```
 ## This opens the file first and closes it after writing to it.
 ##
-## To format data before writing it to a file, you can use [File.write] instead.
+## > Note: to format data before writing it to a file, you can use [File.write] instead.
```

Comment:
On the stylistic side, personally I don't think we need "Note: " on these. I don't think it's really adding anything. 😄

Reply:
Updated.